sshd-core/src/main/java/org/apache/sshd/client/keyverifier/KnownHostsServerKeyVerifier.java

```diff
@@ -536,6 +537,16 @@ protected List<HostEntryPair> findKnownHostEntries(
         return matches;
     }
 
+    /**
+     * Invoked if any matching host entry has a 'revoked' marker
+     *
+     * @param clientSession The {@link ClientSession}
+     * @param remoteAddress The remote host address
+     * @param serverKey     The presented server {@link PublicKey}
+     */
+    protected void handleRevokedKey(ClientSession clientSession, SocketAddress remoteAddress, PublicKey serverKey) {
+    }
```

Comment:
Now PMD is unhappy about this empty block.

Either insert a comment `// By default do nothing`, or move the debug logging from before the call into this body. The latter might make sense; if someone overrides this, he can then decide whether to have that debug logging (call super) or not (don't call super, do something else entirely).